driver-core/src/main/java/com/datastax/driver/core/Cluster.java

```diff
@@ -1259,6 +1263,28 @@ synchronized void init() {
             try {
                 while (true) {
                     try {
+
+                        // create debouncers - at this stage, they are not running yet
+                        QueryOptions queryOptions = configuration.getQueryOptions();
+                        this.nodeListRefreshRequestDebouncer = new EventDebouncer<NodeListRefreshRequest>(
```

Comment:
I think it will be useful, at least for my local stress testing, to track the size of pending events (basically a Gauge over EventDebouncer#eventCount) so I'll add and register a gauge for metrics.  Is it useful enough to include in our code?   Only thing I can think of is to measure impact when tuning with diifferent intervals.